AnagramSolver.BusinessLogic/Classes/Repositories/FileToDatabaseRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AnagramSolver.Contracts.Interfaces;
+using AnagramSolver.EF.DatabaseFirst.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AnagramSolver.BusinessLogic.Classes.Repositories
+{
+    public class FileToDatabaseRepository : IFileToDatabaseRepository
```

Comment:
Consider adding a generic interface `IRepository`:

```
    public interface IRepository<T> where T : class
    {
        IEnumerable<T> GetAll();
        T GetById(object id);
        void Insert(T obj);
        void Update(T obj);
        void Delete(object id);
        void Save();
    }
```

All of the repository classes would implement how all these methods should work.